### ISSUE TITLE
The plugin doesn't work for me

### ISSUE THREAD

**Issue by LevskiWeng** [Xuyuanp/nerdtree-git-plugin#22]
I followed the installation instruction ( I use Vundle as my vim plugin-repositories management tool), and after installing the NERDTree and nerdtree-git-plugin, when I use `:NERDTreeToggle` to show the tree, the NERDTree functions well, but I cannot see the git icon on my local git repository. Do I miss some instruction in my `.vimrc` file ?

The related pieces of my `.vimrc` is as following:

``` vim
call vundle#begin()
Plugin 'gmarik/Vundle.vim'
Plugin 'tpope/vim-fugitive'
Plugin 'bling/vim-airline'
Plugin 'scrooloose/nerdtree'
Plugin 'Xuyuanp/nerdtree-git-plugin'
Plugin 'lsdr/monokai'
call vundle#end()
```

**Comment by Xuyuanp** [comment]
This is a plugin of `nerdtree`, so you should install `nerdtree` too

**Comment by LevskiWeng** [comment]
Well, I've installed `nerdtree` already. In my `.vimrc`, I've added the following:

``` vim
Plugin ‘scrooloose/nerdtree`
```

And after installing the `nerdtree` plugin and the `nerdtree-git-plugin`, the `nerdtree` itself seems to work well while the `nerdtree-git-plugin` doesn't.

After several test, I found the 'nerdtree-git-plugin` works occasionally, but I can't reproduce the scenario.

I'll do some dig to see why the plugin is not working. And I'll post my diagnostic information here.
